### PR TITLE
dev-vcs/git-repo-updater: new package

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
Closes: https://bugs.gentoo.org/694788
Package-Manager: Portage-2.3.76, Repoman-2.3.17
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

### CODE REVIEW

File: dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Easily update multiple git repositories at once"
+HOMEPAGE="https://github.com/earwig/git-repo-updater"
+SRC_URI="https://github.com/earwig/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```
Comment (mgorny) [inline dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild]: `RESTRICT="!test? ( test )"`

**Comment by mgorny** [comment]
Tests are unhappy:
[dev-vcs:git-repo-updater-0.5.1:20191025-101854.log](https://github.com/gentoo/gentoo/files/3771557/dev-vcs.git-repo-updater-0.5.1.20191025-101854.log)

File: dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild (hunk 1 above)
Comment (Alessandro-Barbieri) [reply dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild]: done

File: dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+DEPEND="
+	>=dev-python/colorama-0.3.9[${PYTHON_USEDEP}]
```
[the remark this answers is not in the document]
Comment (Alessandro-Barbieri) [reply dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild]: added

File: dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	>=dev-python/git-python-2.1.8[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		dev-python/pytest[${PYTHON_USEDEP}]
+	)
+"
+
+python_test() {
+	${PYTHON} gitup/test/__init__.py || die
```
[the remark this answers is not in the document]
Comment (Alessandro-Barbieri) [reply dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild]: changed

File: dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild (hunk 2 above)
[the remark this answers is not in the document]
Comment (Alessandro-Barbieri) [reply dev-vcs/git-repo-updater/git-repo-updater-0.5.1.ebuild]: added

**Comment by Alessandro-Barbieri** [comment]
moved to GURU

**Pull request closed without merging**